src/ensembl/src/content/app/browser/track-panel/track-panel-modal/modal-views/TrackPanelBookmarks.test.tsx

```diff
@@ -25,12 +26,25 @@ import { TrackPanelBookmarks } from './TrackPanelBookmarks';
 
 import { PreviouslyViewedObject } from '../../trackPanelState';
 
-jest.mock('react-router-dom', () => ({
-  Link: (props: any) => (
-    <div {...props} className={'link'}>
-      {props.children}
-    </div>
-  )
+jest.mock('react-redux', () => ({
+  useSelector: jest.fn().mockImplementation((selector) => selector()),
+  useDispatch: jest.fn()
+}));
```

Comment:
I would much rather we mocked the relevant part of redux state and used the mock-redux-store to inject it instead of mocking redux and redux hooks.

Reply:
✅